package.json

```diff
@@ -4,6 +4,7 @@
   "private": true,
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/225509972-67f6104f-65ac-4cb6-8056-90f526fa67d2.png)

**[OPTIONAL]**

- [ ] It would be nice if you make your pull request title more descriptive to make your pull request professional 😊 You could have used **React Capstone Finalized** which is more descriptive than **Create tests, add live demo links, and polish details** Your pull request title always tells what is happening in this pull request 👍